### ISSUE TITLE
[Bug]: Post and comment context menu button too small

### ISSUE THREAD

**Issue by ftatateeta** [Memmy-App/memmy#1068]
### Check for open issues

- [ ] I have verified that another issue for this is not open, or it has been closed and has not been fixed.

### Minimal reproducible example

Trying to tap on the context menu buttons for comment or post results in failure half the time because they're too small.

### Expected Behavior

Shouldn't be that hard to tap on the context menu button.

### Version

1.1.78

### App Store or TestFlight?

App Store

### iOS Version

17.x

### Device

iphone x

**Comment by ftatateeta** [comment]
Thanks for the quick fix but did you mean to close the other issue? https://github.com/Memmy-App/memmy/issues/1067

**Comment by gkasdorf** [comment]
@ftatateeta Yea, I fixed that too 👍

**Comment by gkasdorf** [comment]
Oh...you're right. Sorry.

**Comment by ftatateeta** [comment]
Awesome!

**Comment by gkasdorf** [comment]
@ftatateeta Ok, I applied some changes that I feel like are improve this. I'm going to leave this open though and ask for your opinion once the update is available (submitted something to Testflight, but its a new version number so will need to be approved by Apple. Usually fairly quick these days).

Darn users and their different sized fingers 🤣

**Comment by gkasdorf** [comment]
Admittedly I was slightly lazy here and tried doing it with an invisible SVG. Adding padding will increase spacing and sort of mess up the positioning of elements. I plan to go through and take time making every specific UI piece look better, so at that time I’d definitely try to take the ellipsis into account (the issue is the height of it is obviously rather small)But if this doesn’t feel right we can just go ahead and do something about it now 👍GavinSent from my iPhoneOn Nov 6, 2023, at 17:08, ftatateeta ***@***.***> wrote:﻿
Thanks I'll take a look later when I get a chance. I thought we could add invisible padding to increase the button area without increasing the icon size as well. I think that's how apps usually do it since the actual button is usually bigger than the visual icon size to make it easier to tap. Anyway I don't know if that's necessary I'll give it a try.

**Comment by ftatateeta** [comment]
I just tried it it's a lot better now. Thank you!
